centreon/tests/e2e/features/Cloud-notifications/05-notification-get.feature

```diff
@@ -0,0 +1,27 @@
+@ignore
+@REQ_MON-24183
+Feature: Get notification rules
+  As a Centreon user with access to the notification rules page
+  I need to see the list of notification rules
+  So that I can organize my alerts on my monitoring system
+
+  Background:
+    Given a user with access to the notification rules page
+    And the user is on the notification rules page
+    And There is at least one minimum notification rule
```

Comment:
it doesn't make sense to me, since you want to test the listing with pagination features, you should have more than one notification created